src/H5FDsubfiling/H5subfiling_common.c

```diff
@@ -2074,8 +2063,13 @@ record_fid_to_subfile(uint64_t file_id, int64_t subfile_context_id, int *next_in
             *next_index = index;
         }
 
-        sf_open_file_map[index].file_id         = file_id;
-        sf_open_file_map[index++].sf_context_id = subfile_context_id;
+        sf_open_file_map[index].file_id       = file_id;
+        sf_open_file_map[index].sf_context_id = subfile_context_id;
```

Comment:
Was the previous 'index++' a bug?

Reply:
It was just unnecessary since it's at the end of the function. Likely I copy-pasted from a loop at some point.